File: sqlUpdate.py
import MySQLdb
db = MySQLdb.connect(host = "localhost",user = "username",passwd="password",
db="students")
print("Hello World!")
#cursor

cur = db.cursor(MySQLdb.cursors.DictCursor)

sql = "UPDATE students SET gradeLvl=9 WHERE id=4)" # "SELECT * from students ORDER BY name"

cur.execute(sql)
rows = cur.fetchall()
#send code
db.autocommit(True)
cur.close()
db.close()

File: sqlUpdate.php
<?php
$servername = "localhost";
$username = "username";
$password = "password";
$dbname = "students";

// Create connection
$conn = new mysqli($servername, $username, $password, $dbname);
// Check connection
if ($conn->connect_error) {
   die("Connection failed: " . $conn->connect_error);
}

$sql = "UPDATE students SET gradeLvl=12 WHERE id=3";

if ($conn->query($sql) === TRUE) {
   echo "New record created successfully";
} else {
   echo "Error: " . $sql . "<br>" . $conn->error;
}

$conn->close();
?>
